plugins/@grouparoo/mailjet/src/lib/export/exportProfile.ts

```diff
@@ -0,0 +1,155 @@
+import {
+  ExportProfilePluginMethod,
+  Errors,
+  SimpleAppOptions,
+  DestinationSyncOperations,
+} from "@grouparoo/core";
+import { connect } from "../connect";
+import { MailjetClient } from "../client";
+import { addToList, removeFromList } from "./listMethods";
+
+const deleteContactOrClearGroups = async (
+  client: MailjetClient,
+  syncOperations: DestinationSyncOperations,
+  contact: any,
+  groups: string[]
+) => {
+  if (syncOperations.delete) {
+    await client.deleteContact(contact.ID);
+  } else {
+    if (syncOperations.update) {
+      // clear groups
+      for (const group of groups) {
+        await removeFromList(client, contact.Email, group);
+      }
```

Comment:
Looks like we could potentially be making a lot of API Calls... did you run into any rate limits?